lib/protocols/encoder_structs/base64.ex

```diff
@@ -0,0 +1,3 @@
+defmodule EaRestaurantDataLoader.Base64 do
```

Comment:
Shouldn't the namespace be `EaRestaurantDataLoader.Lib.Protocols.PasswordEncoder.Base64`?

Could you make sure the namespace of you project is right for all files? (including the test files)

Reply:
Done. It was fixed.